tests/utils.cpp

```diff
@@ -0,0 +1,13 @@
+#include "utils.h"
+
+#include "faabric_utils.h"
+#include <faasm/emulator.h>
+
+namespace tests {
+void cleanCppTests()
+{
+    cleanFaabric();
+
+    setEmulatorUser("tests");
+}
```

Comment:
@csegarragonz you can see here that `cleanCppTests` is essentially just a wrapper around `cleanFaabric` but also resetting the emulator user.

We have functions like this in Faasm (`cleanSystem`) and Faabric (`cleanFaabric`). Their responsibility is to clean up anything that might allow state to bleed across tests. This is optional, to be called only at the start of tests that interact with those stateful parts of the system. We could call it on every test but it's non-trivial as it might be interacting with Redis and performing other time-consuming operations.